package.json

```diff
@@ -53,11 +60,12 @@
     "css-loader": "^0.23.1",
     "eslint": "^5.4.0",
     "eslint-config-airbnb": "^17.1.0",
+    "eslint-config-prettier": "^3.0.1",
```

Comment:
I believe my VSCode automatically reordered this line to be in alphabetical order on save, it's also responsible for reformatting the `keywords` array a few lines above.

Reply:
ah, good to know